dev/DataSource/Mapping/LanguageExtensiveMapping.php

```diff
@@ -23,25 +24,26 @@ public static function url(): string
         return 'https://iso639-3.sil.org/sites/iso639-3/files/downloads/iso-639-3_Latin1.tab';
     }
 
+    /** @throws DomElementNotFoundException */
     public static function toDataSet(Client $client, Crawler $crawler): array
     {
         $dataSet = [];
         foreach (explode("\n", $client->getPageSource()) as $lineNumber => $line) {
-            if ($lineNumber === 0) {
+            if ($lineNumber === 0 || $line === '</pre></body></html>') {
```

Comment:
@PrinsFrank Does the `.tab` file HTML in it?

Reply:
I don't know why, this maybe related to the panther implementation that the browser goes in quirks mode. I need to thoroughly look at this, but I was in a bit of a rush before.